src/tjs.h

```diff
@@ -34,8 +34,9 @@
 
 typedef struct TJSRuntime TJSRuntime;
 
-TJSRuntime *TJS_NewRuntime(void);
+TJSRuntime *TJS_NewRuntime(RunOption *option);
 TJSRuntime *TJS_NewRuntime2(bool is_worker);
+TJSRuntime *TJS_NewRuntimeByOptions(bool is_worker, RunOption *option);
```

Comment:
Please leave `TJSRuntime *TJS_NewRuntime(void);` alone, taking the default values.

Reply:
I plan to change it like this, what do you think:

``` c
TJSRuntime *TJS_NewRuntime(void);
TJSRuntime *TJS_NewRuntimeWorker(void);
TJSRuntime *TJS_NewRuntimeOption(bool is_worker, TJSRunOptions *option);
```

Reply:
Sounds good!